src/Security/User/NullUser.php

```diff
@@ -4,15 +4,25 @@
 
 namespace League\Bundle\OAuth2ServerBundle\Security\User;
 
+use League\OAuth2\Server\Entities\UserEntityInterface;
 use Symfony\Component\Security\Core\User\UserInterface;
 
 /**
  * @psalm-immutable
  *
  * @author Mathias Arlaud <mathias.arlaud@gmail.com>
+ * @author Aurélien Pillevesse <aurelienpillevesse@hotmail.fr>
  */
-final class NullUser implements UserInterface
+final class NullUser implements UserInterface, UserEntityInterface
 {
+    /**
+     * @psalm-mutation-free
+     */
+    public function getIdentifier(): string
```

Comment:
Shouldn't we respect the interface declaration with no return type and return a null value? https://github.com/thephpleague/oauth2-server/blob/master/src/Entities/UserEntityInterface.php#L19C7-L19C7